packages/office-addin-dev-settings/src/publish.ts

```diff
@@ -3,24 +3,66 @@
 
 import * as childProcess from "child_process";
 import * as fs from "fs";
+import * as path from "path";
 
 /* global console */
 
-export async function registerWithTeams(zipPath: string): Promise<void> {
+export async function registerWithTeams(zipPath: string): Promise<string> {
   return new Promise((resolve, reject) => {
     if (zipPath.endsWith(".zip") && fs.existsSync(zipPath)) {
-      const command = `npx @microsoft/teamsfx-cli provision manifest --file-path ${zipPath}`;
+      const sideloadCommand = `npx @microsoft/teamsfx-cli@1.0.6-alpha.18494658a.0 m365 sideloading --file-path ${zipPath}`;
```

Comment:
I've been having trouble getting the right version of the package to run since teamsfx-cli a pre-release and could be a dependency in any of the different layers (dev-settings, debugging, template) and npm could pull if from different places.  For pre-release we need to be specific . . . after there is a latest version supported then we can remove the version.